device/mellanox/x86_64-mlnx_msn3800-r0/Mellanox-SN3800-D24C52/sai.profile

```diff
@@ -0,0 +1 @@
+SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/sai_3800.xml
```

Comment:
Kindly keep the name as sai_3800_24x50g_52x100g as that's a convention followed for other SKU's as well. You can check this . It can be used to identify the file in syncd docker

Reply:
addressed in the next commit